features/storage/hostpath.feature

```diff
@@ -10,12 +10,14 @@ Feature: Storage of Hostpath plugin testing
       | ["spec"]["hostPath"]["path"] | /etc/origin/hostpath/<%= project.name %> |
       | ["spec"]["hostPath"]["type"] | DirectoryOrCreate                        |
       | ["spec"]["accessModes"][0]   | <access_mode>                            |
+      | ["spec"]["storageClassName"] | sc-<%= project.name %>                   |
       | ["spec"]["persistentVolumeReclaimPolicy"] | <reclaim_policy>            |
```

Comment:
There is not align

Reply:
@chao007 hmmm...alignment looks fine.

Reply:
@chao007 The alignment issue on line 14 is not related to this PR. Anyway, i will fix it.

Reply:
Now the alignment issue is fixed. The real change of adding storageClassName is not so obviously now.